### PR TITLE
Pages and categories can be hidden or published (with schema migrations)

### PR DESCRIPTION
**Pull request by mutantmonkey**
Do not merge this yet, it needs to be tested. This won't work with SQLite at all unfortunately, which means I'll need to convert my local dev instance to use a sane database first, or we could set up a staging instance.

This obsoletes #168.

### CODE REVIEW

File: wuvt/models.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class Page(db.Model):
     published = db.Column(db.Boolean, default=True, nullable=False)
 
 
-    def __init__(self, name, slug, content, published=True, menu=None):
+    def __init__(self, name, slug, content, published, menu=None):
```
Comment (mutantmonkey) [inline wuvt/models.py]: @bkodres93 What is the reason for this change?

File: create.py [hunk 2]
```diff
@@ -16,9 +16,9 @@
 
 
 
-cats = [Category("News", "news"), Category("Sports", "sports"),
-        Category("Weather", "weather"),
-        Category("Woove", "woove")]
+cats = [Category("News", "news", True), Category("Sports", "sports", True),
```
Comment (mutantmonkey) [inline create.py]: If all of these are True, why not set that to be the default so no change is needed here? See comment below...

**Comment by mutantmonkey** [comment]
This is another example of GitHub failing to have sufficient code review functionality; there's no ability to have pull requests that are dependent upon each other. On second thought, it might make more sense to split these back out, but it's not like I can add commits to @bkodres93's pull request either...

File: wuvt/models.py (hunk 1 above)
Comment (bkodres93) [inline wuvt/models.py]: I feel it makes sense to have to check the box to actually publish something, rather than having it published by default. Safety first.

File: create.py (hunk 2 above)
Comment (bkodres93) [inline create.py]: above.

**Comment by mutantmonkey** [comment]
It would be helpful if someone could test this since I haven't gotten around to it yet.
